api/interfaces/pinia.PiniaCustomStateProperties.md

```diff
@@ -4,16 +4,16 @@ editLinks: false
 sidebarDepth: 3
 ---
 
-[API Documentation](../index.md) / [pinia](../modules/pinia.md) / PiniaCustomStateProperties
+[Documentación de la API](../index.md) / [pinia](../modules/pinia.md) / PiniaCustomStateProperties
 
-# Interface: PiniaCustomStateProperties<S\>
+# Interfaz: PiniaCustomStateProperties<S\> {#interfaz-piniacustomstateproperties-s}
 
 [pinia](../modules/pinia.md).PiniaCustomStateProperties
 
-Properties that are added to every `store.$state` by `pinia.use()`.
+Propiedades que se agregan a cada `store.$state` por `pinia.use()`.
```

Comment:
Creo que esto se podría traducir mejor como "Propiedades que se añaden a cada `store.$state` mediante `pinia.use()`." ¿Cómo lo ves?

Reply:
Me parece bien, se lee más fluido de esa manera ✨ changing it